uuagc/trunk/src/JSON.hs

```diff
@@ -0,0 +1,41 @@
+module JSON where
```

Comment:
I'd prefer to remove this module and depend on [`aeson`](http://hackage.haskell.org/package/aeson) instead, which is the standard for JSON encoding of Haskell data types.

Reply:
Ah, I chose to do it this way to add minimal dependencies to UUAGC, but if you're fine with Aeson then that also works.